### DISCUSSION TITLE
Screen without login

### DISCUSSION THREAD

**Discussion by p4bl1t0** [orchidsoftware/platform#1848] [Q&A]
Is it possible to create screen (using the orchid layout system) that doesn't require the user to be logged in?

**Comment by tabuna** [comment]
Hi, just remove the `platform` middleware in the config file (don't forget to clear the cache):

https://github.com/orchidsoftware/platform/blob/53518188589d02d4c61b90c99fd167c6cf9c4596/config/platform.php#L50-L52

**Comment by tabuna** [comment]
Hi, just remove the `platform` middleware in the config file (don't forget to clear the cache):

https://github.com/orchidsoftware/platform/blob/53518188589d02d4c61b90c99fd167c6cf9c4596/config/platform.php#L50-L52

**Comment by limogin** [comment]
But it is possible to disable it only in some screens? For example, to display a registration form

**Comment by p4bl1t0** [comment]
```
Route::domain((string) config('platform.domain'))
->prefix(Dashboard::prefix('/'))
->as('platform.')
->middleware(config('platform.middleware.public'))
->group(base_path('routes/custom_public.php'));
```

In the RouteServiceProvider.php I just added my own route. The routes in custom_public will get all the orchid props and theme without being login
